### DISCUSSION TITLE
privacytests.org results

### DISCUSSION THREAD

**Discussion by arthuredelstein** [bromite/bromite#1689] [Show and tell]
Hi everyone -- I am pleased to share the results of privacy tests for Bromite:

https://privacytests.org/android.html

Bromite is one of the better browsers for privacy on Android -- the toughest competition comes from Brave and Tor Browser.

I'd be happy to answer any questions about these tests!

**Comment by Mr-Administrator** [comment]
Keep up the good work!

**Reply by arthuredelstein** [reply]
Thanks!

**Comment by uazo** [comment]
interesting, congratulations.
I don't know how particularly reliable it can be regarding privacy because there are probably no references of what the browser itself sends without the user's knowledge (which with bromite should be nothing, apart from the update, it can also be disabled), but it is a good basis to work on, too bad it is not present how do you detect the results (or maybe I don't see it) could be useful for setting up some tests.

**Reply by arthuredelstein** [reply]
> interesting, congratulations. I don't know how particularly reliable it can be regarding privacy because there are probably no references of what the browser itself sends without the user's knowledge (which with bromite should be nothing, apart from the update, it can also be disabled), but it is a good basis to work on, too bad it is not present how do you detect the results (or maybe I don't see it) could be useful for setting up some tests.

Thanks! The full source code of tests is available at https://github.com/arthuredelstein/privacytests.org. You can also click on any individual test result to see the test code.

**Comment by mrx23dot** [comment]
Any plans to make bromite the greenest privacy respecting browser?

**Comment by NineOutOfTen** [comment]
I like your site. However, I wish Bromite would be updated more often. That makes it difficult to recommend or use. Unfortunately, a few well known privacy websites have dropped it from their lists of recommended browsers citing infrequent updates.
